public/main.css

```diff
@@ -3,4 +3,56 @@
 html {
 	background-color: #036;
 	
+}
+
+.content-area {
+    width: 95%;
+    height: auto;
+    margin: 0 2.5% auto 2.5%;
+    padding: 0;
+}
+
+.page-section {
+    background-color: #FFF;
+    margin: 1em auto auto auto;
+    padding: 2.5%;
+}
+
+.page-section h1 {
+    margin: 0;
+    padding: 0;
+}
+
+.welcome-text {
+    width: 100%;
+    display: inline-block;
+    vertical-align: top;
+}
+
+.welcome-image{
+    width: 100%;
+    display: inline-block;
+    right: 0;
+    margin: 0;
+    vertical-align: top;
+}
+
+.welcome-image img {
+    width: 100%;
+}
+
+@media (min-width: 40rem) {
```

Comment:
Denne gjør at alt som ligger i denne blokken kun gjelder dersom størrelsen på nettleseren (og skjermen) er 40rem eller bredere. Den kalles en media query, men jeg vet ikke helt hvorfor. Dette gjør at nettsiden kan se forskjellig ut på for eksempel telefoner og PCer.

Rem er en enhet som er relativ til font-størrelsen på rot-elementet (altså html). 1rem = høyden på en M.
Du kan lese om CSS-enheter her hvis du vil https://www.w3schools.com/CSSref/css_units.asp